dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -661,11 +661,53 @@ const textCardHeadline = (format: ArticleFormat): string => {
 					return WHITE;
 			}
 		default:
-			return BLACK;
+			switch (format.theme) {
```

Comment:
I think you want to make changes in [`textCardKicker`](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/lib/decidePalette.ts#L676-L718) method instead!